packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -320,6 +320,31 @@ const PRESETS = [
         assets: ['translations'],
         private: true
     },
+    {
+        id: 'retail-react-app-bug-bounty',
+        name: 'Retail React App Bug Bounty Project',
+        description: '',
+        templateSource: {
+            type: TEMPLATE_SOURCE_NPM,
+            id: '@salesforce/retail-react-app'
+        },
+        questions: [...EXTENSIBILITY_QUESTIONS, ...HYBRID_QUESTIONS, ...RETAIL_REACT_APP_QUESTIONS],
```

Comment:
Do we need  `HYBRID_QUESTIONS` in this preset?

```suggestion
        questions: [...EXTENSIBILITY_QUESTIONS, ...RETAIL_REACT_APP_QUESTIONS],
```